lib/peer_manager/modules/nrf_ble_lesc.c

```diff
@@ -144,6 +151,24 @@ uint32_t nrf_ble_lesc_keypair_generate(void)
 		LOG_ERR("psa_export_public_key() returned status %d", status);
 		return NRF_ERROR_INTERNAL;
 	}
+
+#if defined(CONFIG_PM_LESC_PRIVATE_KEY_EXPORT)
+	uint8_t priv_key[ECC_PRIV_KEY_EXPORT_SIZE];
```

Comment:
I would also put a big log disclaimer. Putting it in the Kconfig description is not visible enough.